zipline/pipeline/factors/factor.py

```diff
@@ -1229,9 +1236,86 @@ def compute(self, today, assets, out, data):
         # MedianValue.
         median_close10 = MedianValue([USEquityPricing.close], window_length=10)
         median_low15 = MedianValue([USEquityPricing.low], window_length=15)
+
+    A CustomFactor with multiple outputs:
+
+    .. code-block:: python
+
+        class MultipleOutputs(CustomFactor):
+            inputs = [USEquityPricing.close]
+            outputs = ['alpha', 'beta']
+            window_length = N
+
+            def compute(self, today, assets, out, close):
+                computed_alpha, computed_beta = some_function(close)
+                out.alpha[:] = computed_alpha
+                out.beta[:] = computed_beta
+
+        # Each output is returned as its own Factor upon instantiation.
+        alpha, beta = MultipleOutputs()
+
+        # Equivalently, we can create a single factor instance and access each
+        # output as an attribute of that instance.
+        multiple_outputs = MultipleOutputs()
+        alpha = multiple_outputs.alpha
+        beta = multiple_outputs.beta
+
+    Note: If a CustomFactor has multiple outputs, all outputs must be of the
```

Comment:
Grammar nit: "must be of the same" is a bit awkward.  "must have the same" is cleaner IMO.